frontend/src/components/ScoreBadge.tsx

```diff
@@ -3,11 +3,20 @@ import classNames from "classnames"
 
 import styles from "./ScoreBadge.module.scss"
 
+function calculateScorePercent(score: number, maxScore: number) {
+    if (score > maxScore) {
+        return 0
```

Comment:
```suggestion
function calculateScorePercent(score: number, maxScore: number): string {
    if (score > maxScore) {
        return "0"
```

Reply:
I disagree that this should be returning a string

Reply:
overruled